### PR TITLE
Improved Error handling

### PR DESCRIPTION
**Pull request by yaakovi**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27868

## Description
* convert to package.
* improved error handling.
* added unit tests

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
Looks good, but very volatile, as this script is used in each and every customer env.
Is there a customer or 2 we can share with before releasing?

File: Packs/CommonScripts/Scripts/RunPollingCommand/RunPollingCommand_test.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        {'ids': 'a,b,c', 'arg1': 'value1', 'arg2': 'value2'},
+    ),
+    # extra args
+    (
+        ('ids', ['a', 'b', 'c'], u'arg1, arg2,arg3', [u'value1', u'value2', u'value3']),
+        {'ids': 'a,b,c', 'arg1': 'value1', 'arg2': 'value2', 'arg3': 'value3'},
+    ),
+]
+
+
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/RunPollingCommand/RunPollingCommand_test.py:38]: can we add a few more cases for the `additional_arg_names`, `additional_arg_values`?

**Review by yuvalbenshalom — APPROVED** [review]
looks ok, but tests did not run.
@bakatzir since this is covered by a number of test playbooks that are running it(thru generic polling), I think it will be ok if we will see dose tests run and pass.

**Comment by yaakovi** [comment]
build is too long (exceed 5 hours) so it will keep on crashing

**Pull request merged**